src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -472,6 +472,12 @@ class MFEMSidreDataCollection : public mfem::DataCollection
    */
   virtual void Load(int cycle_ = 0)
   {
+  #ifndef AXOM_USE_HDF5
+    SLIC_ERROR(
+      "MFEMSidreDataCollection::Load(<cycle>) is only implemented for the "
+      "'sidre_hdf5' protocol");
+  #endif
```

Comment:
It was somewhat surprising that the `MFEMSidreDataCollection::load(<cycle>)` didn't support other protocols. 
(Although the doxygen already noted this).

I added this error message so it's easier for people to see why this doesn't work. I don't think it would be too difficult to make this work with other protocols, but I also am not aware of any need/desire for this at the moment, so I'm planning to leave this as is.